docs/code/DataViewRowCursor.md

```diff
@@ -1,33 +1,32 @@
 ﻿# `DataViewRowCursor` Notes
 
-This document includes some more in depth notes on some expert topics for
+This document includes some more in-depth notes on some expert topics for
 `DataViewRow` and `DataViewRowCursor` derived classes.
 
 ## `Batch`
 
 Multiple cursors can be returned through a method like
 `IDataView.GetRowCursorSet`. Operations can happen on top of these cursors --
 most commonly, transforms creating new cursors on top of them for  parallel
-evaluation of a data pipeline. But the question is, if you need to "recombine"
-them into a sequence again, how do to it? The `Batch` property is the
-mechanism by which the data from these multiple cursors returned by
+evaluation of a data pipeline. But the question is if you need to "recombine"
+them into a sequence again, how to do it? The `Batch` property is the
+mechanism by which the data from these multiple cursors, returned by
 `IDataView.GetRowCursorSet` can be reconciled into a single, cohesive,
 sequence.
 
-The question might be, why recombine. This can be done for several reasons: we
+The question might be, why recombine? This can be done for several reasons: we
 may want repeatability and determinism in such a way that requires we view the
 rows in a simple sequence, or the cursor may be stateful in some way that
 precludes partitioning it, or some other consideration. And, since a core
-`IDataView` design principle is repeatability, we now have a problem of how to
-reconcile those separate partitioning.
+`IDataView` design principle is repeatability, we now have a problem with how to
+reconcile those separate partitions.
 
 Incidentally, for those working on the ML.NET codebase, there is an internal
 method `DataViewUtils.ConsolidateGeneric` utility method to perform this
-function. It may be helpful to understand how it works intuitively, so that we
+function. It may be helpful to understand how it works intuitively so that we
 can understand `Batch`'s requirements: when we reconcile the outputs of
 multiple cursors, the consolidator will take the set of cursors. It will find
-the one with the "lowest" `Batch` ID. (This must be uniquely determined: that
-is, no two cursors should ever return the same `Batch` value.) It will iterate
+the one with the "lowest" `Batch` ID. (This must be uniquely determined: i.e. no two cursors should ever return the same `Batch` value.) It will iterate
```

Comment:
Just omit the `i.e.` and maintain line length.

```suggestion
the one with the "lowest" `Batch` ID. (This must be uniquely determined: 
no two cursors should ever return the same `Batch` value.) It will iterate
```